pkg/api/graph/interfaces.go

```diff
@@ -107,5 +105,5 @@ func (m ByKey) Less(i, j int) bool {
 type Suggestion string
 
 func (s Suggestion) String() string {
-	return fmt.Sprintf("try: %s", string(s))
```

Comment:
> That's 100% incorrect - this is the most important info we have ever
> returned in oc status, because it is what every user who tries to run
> openshift with their custom image the first time hits.  If we can't explain
> it here, we should scrap the project and start over.

If it appears more than once it's so long and verbose, that no user will ever read it.  I knew what you were trying to say and I barely kept reading long enough to make to the "what should I do next" step.  If you give a copiable command for `oc logs -c container -p pods/POD` as the very first thing and then maybe the explanation root permissions, I think this would be much more effective.